vllm/entrypoints/openai/api_server.py

```diff
@@ -523,6 +524,18 @@ async def detokenize(request: DetokenizeRequest, raw_request: Request):
     assert_never(generator)
 
 
+def maybe_register_tokenizer_info_endpoint(args):
+    """Conditionally register the tokenizer info endpoint if enabled."""
+    if getattr(args, 'enable_tokenizer_info_endpoint', False):
+        @router.get("/get_tokenizer_info")
```

Comment:
```suggestion
        @router.get("/tokenizer_info")
```

`get_` is redundant because this is a GET endpoint already